src/desktop/analytics/main_layout.js

```diff
@@ -3,27 +3,66 @@
 // or any other actions that occur on each page.
 //
 
+import { data as sd } from 'sharify'
+
 // Track pageview
-analytics.page({path: location.pathname}, {integrations: {'Marketo': false}})
+analytics.page(
+  { path: location.pathname },
+  { integrations: { Marketo: false } }
+)
+
+window.addEventListener('load', function() {
+  if (
```

Comment:
#minor maybe this guard could be placed outside the event listener (to avoid it being set at all)?